### PR TITLE
Commerce 9935

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-9935

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/SkuUtil.java [hunk 1]
```diff
@@ -197,9 +210,35 @@ private static String _getOptions(Sku sku) {
 		for (SkuOption skuOption : skuOptions) {
 			jsonArray.put(
 				JSONUtil.put(
-					"key", skuOption.getKey()
+					"key",
+					() -> {
+						if (skuOption.getKey() == null) {
```
Comment (alessiorendina) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/SkuUtil.java:215]: Can't we use `Validator.isNull(skuOption.getKey())` instead?

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/SkuUtil.java [hunk 2]
```diff
@@ -197,9 +210,35 @@ private static String _getOptions(Sku sku) {
...
+						}
+
+						return skuOption.getKey();
+					}
 				).put(
-					"value", JSONUtil.put(skuOption.getValue())
+					"value",
+					JSONUtil.put(
+						() -> {
+							if (skuOption.getValue() == null) {
```
Comment (alessiorendina) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/SkuUtil.java:230]: Same here

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra It seems good to me, just a small change. Thank you!

**Review by alessiorendina — APPROVED** [review]

**Pull request closed without merging**